client/src/components/AboutPage/AboutFeedbackSection/AboutFeedbackSection.tsx

```diff
@@ -19,7 +19,9 @@ const AboutFeedbackSection: FunctionComponent = () => (
         <h2>Leave A Feedback</h2>
         <p>Liked our service or maybe were dissatisfied with it?</p>
         <Link to="/feedback">
-          <Button color="white">Tell Us About It</Button>
+          <Button type="button" ariaLabel="Go to feedback page" color="white">
```

Comment:
you can extend `ButtonProps` interface with `ButtonHTMLAttributes<HTMLButtonElement>` and define attributes the same way as for plain html button